instrumentation/metric/rate.go

```diff
@@ -53,3 +56,19 @@ func (r *Rate) Measure(eventCount int64) {
 
 	r.runningSum += eventCount
 }
+
+func (r *Rate) Reset() {
```

Comment:
we need to understand the relationship between the tick interval (in ` func Measure`) and the polling interval. Does it really make sense to reset the rate? I'm not sure anymore - maybe look at how other people use the `MovingAverage` library.

also, if we DO reset, do we want to reset `runningSum` as well?

Reply:
I decided not to reset at all.